packages/text-field/theme/lumo/vaadin-input-field-shared-styles.js

```diff
@@ -73,33 +64,6 @@ registerStyles(
       opacity: 0.5;
     }
 
-    [part='input-field'] {
-      border-radius: var(--lumo-border-radius-m);
-      background-color: var(--lumo-contrast-10pct);
-      padding: 0 calc(0.375em + var(--lumo-border-radius-m) / 4 - 1px);
-      font-weight: 500;
-      line-height: 1;
-      position: relative;
-      cursor: text;
-      box-sizing: border-box;
-    }
```

Comment:
This CSS is duplicated here:

https://github.com/vaadin/web-components/blob/91bfb8acc231ec7747bbebbfe2add5b83e647e00/packages/input-container/theme/lumo/vaadin-input-container-styles.js#L10-L22